spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/ColorJitterSpec.scala

```diff
@@ -30,13 +30,10 @@ import com.intel.analytics.bigdl.utils.RandomGenerator
 
 @com.intel.analytics.bigdl.tags.Serial
 class ColorJitterSpec extends FlatSpec with BeforeAndAfter with Matchers {
-  before {
+  "A ColorJitter" should "blend image correctly" in {
     if (!TH.hasTorch()) {
```

Comment:
This should be move into the test case from before. Or it will cause test suite failure